DigitalLearningSolutions.Web/Styles/index.scss

```diff
@@ -15,18 +15,33 @@ ul > li > ul > li {
 .js-enabled .non-js-only {
   display: none;
 }
+

```

Comment:
Ah - this is done in https://github.com/TechnologyEnhancedLearning/DLSV2/pull/378/files too, so worth coordinating with @JBsoftwire

Reply:
Replace the version I have here with the version from that file.

Reply:
Ah - can you pull the latest master for this, should be merged now? 🙏